src/main/java/shedar/mods/ic2/nuclearcontrol/items/ItemCard55Reactor.java

```diff
@@ -59,20 +63,83 @@ public CardState update(TileEntity panel, ICardWrapper card, int range) {
 			return CardState.NO_TARGET;
 		}
 	}
+	
+	private <T> T getMethode(Class<T> par1, Object instance, String functionName)
```

Comment:
Well: Methode = German, Method = Englisch
So from spelling its right... You can change it into englisch^^"
And before it was called: getFunction.